app/views/spree/orders/_summary.html.haml

```diff
@@ -26,36 +28,37 @@
         %td.text-right.total{"data-hook" => "order_item_total"}
           %span= item.display_amount_with_adjustments.to_html
 
-  %tfoot
-    #subtotal{"data-hook" => "order_details_subtotal"}
-      %tr#subtotal-row.total
-        %td.text-right{colspan: "3"}
-          %strong
-            = t :order_produce
-        %td.text-right.total
-          %span= display_checkout_subtotal(order)
-
-    #order-charges{"data-hook" => "order_details_adjustments"}
-      - checkout_adjustments_for(order, exclude: [:line_item]).reject{ |a| a.amount == 0 }.reverse_each do |adjustment|
-        %tr.total
-          %td.text-right{:colspan => "3"}
+  - if display_footer
+    %tfoot
```

Comment:
I would move the conditional section to a partial.
```suggestion
  = render "totals_footer" if display_footer
```

Reply:
Ok, will do!